data/hai/hai missions.txt

```diff
@@ -2099,3 +2099,213 @@ mission "Hai Pilgrimage"
+				accept
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+
+mission "Acorn Delights: Part 2"
+	landing
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	source "Stonebreak"
+	destination "Cloudfire"
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		has "Acorn Delights: Part 1: done"
+	on offer
+		log "Took a Hai lady named Eeeya to Stonebreak to help find her granddaughter, who is named Anoo."
+		conversation
+			`You spend several hours flying Eeeya between the massive communities of Stonebreak. She checks every homeless shelter, volunteer ward, women's shelter, and low-income hospital. The story is always the same: Anoo hasn't been seen in years, though some remember the energetic little girl in her youth before her mother left.`
+			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Since it's obvious she's not here, let's check on <destination> - she could be in military."`
+				accept
+	on visit
+		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
+
+mission "Acorn Delights: Part 3"
+	landing
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	clearance `After hearing the name Eeeya, the Darkcloak flight controller excitedly responds, "Acorn Delights!" and you're granted landing clearance.`
+	source "Cloudfire"
+	destination "Darkcloak"
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		has "Acorn Delights: Part 2: done"
+	on offer
+		conversation
+			`Eeeya takes you to the visitor's office and asks to see the Cloudfire military commander. Upon refusal, Eeeya throws a fit; after an hour of her panic they agree to let her speak with the commander.`
+			`	After a few minutes, she comes back looking despondent, and you lead her back to <ship>.`
+			`	"They said she doesn't work for them. I think they know where she is, but they won't tell me anything.`
+			`	"There's one more place we can try, but I don't think you're going to like it. She may have joined Unfettered Hai."`
+			label questions
+			choice
+				`	"Why didn't you mention this earlier?"`
+					goto mention
+				`	"Why would a doctor join a band of warmongers?"`
+					goto warmonger
+				`	"As the honorable Captain <first> <last>, I say let's go find her!"`
+					goto accept
+				`	"I'd be happy to help."`
+					goto accept
+				`	"Unfettered space is too dangerous for me. You will have to find another captain."`
+					decline
+			label mention
+			`	"I was afraid it would scare you away. Humans are terrified of our ships, and the Unfettered are deadly warriors. Besides, the other places were more likely. Her mother's family does have a tradition of joining the Unfettered, but I didn't think that would happen to her."`
```

Comment:
```suggestion
			`	"I was afraid it would scare you away. Humans are scared of our ships, and the Unfettered are deadly warriors. Besides, the other places were more likely. Many of her mother's family have joined the Unfettered in the past, but I didn't think that would happen to her."`
```